### ISSUE TITLE
getting started with federated wiki

### ISSUE THREAD

**Issue by WardCunningham** [WardCunningham/remodeling#6]
We'll be working on issue #3, integration with federated wiki, in a month or two. If you're interested in participating in the read-write version of the original wiki you will have to learn some new things. It is different but with many benefits that are hard to recognize when you start.

For work on this project I am happy to host related pages in a wiki farm I run. You will need to make yourself a subdomain and try authoring there. Here is a link. Change the subdomain to your name or handle. Then follow the second page instructions. Q&A welcome as comments here.

http://yourname.tries.fed.wiki/view/welcome-visitors/hello.ward.bay.wiki.org/field-guide-to-the-federation

We'll be posting more useful links as we start the integration. If you are still interested and think you can work the new wiki, then post a link to your subdomain in the comments here or email me directly. We'll assemble into a "pod", wiki's name for an online workgroup, once we get going.

![image](https://cloud.githubusercontent.com/assets/12127/19486427/34b9cf02-9513-11e6-8745-a6896704f506.png)

**Comment by gutierlf** [comment]
The "You're New Here" page says

> 
Claim the site. See where it says, Sign in with your Email? Click that button. Login mechanisms vary from site to site so follow the instructions that pop up.

But I find no button for signing in.

**Comment by WardCunningham** [comment]
Good catch. We've moved on from Mozilla Persona which closes at the end of next month. I've revised the field guide to say:
![image](https://cloud.githubusercontent.com/assets/12127/19498343/34b748a4-9545-11e6-9739-0d88a94771f2.png)

The `tries.fed.wiki` farm is configured to use Google for identity. You will need a Google id to claim a site there.

**Comment by anoxic** [comment]
is there an api for the federated wiki system? i'd like to use an existing wiki i built, but it would be interesting to work in a connection with the federated wiki system.

**Comment by WardCunningham** [comment]
GET slug.json

Returns a page like this:
http://ward.asia.wiki.org/json-schema.html

You should google around a bit before asking too many questions here.

**Comment by DustinWehr** [comment]
pdf link on names-of-things page. link is to http://asia.wiki.org/things.pdf]

@WardCunningham kinda nervous to say anything here after your response to @anoxic.

**Comment by DustinWehr** [comment]
Couldn't find instructions anywhere on how to close a page. A subsection of How To Read Content on how-to-wiki would be a good place for that.
